test-junit5/src/main/java/io/micronaut/test/extensions/junit5/MicronautJunit5Extension.java

```diff
@@ -89,6 +95,20 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
                 Object testInstance = extensionContext.getRequiredTestInstance();
                 applicationContext.inject(testInstance);
             }
+
```

Comment:
All this code can be in the superclass.

Reply:
I ended up pushing it all into a static helper method